src/ast/mod.rs

```diff
@@ -4376,7 +4410,31 @@ impl fmt::Display for Statement {
                     full = if *full { "FULL " } else { "" },
                 )?;
                 if let Some(db_name) = db_name {
-                    write!(f, " FROM {db_name}")?;
+                    let keyword = match &db_name_keyword {
+                        Some(Keyword::FROM) => "FROM",
+                        Some(Keyword::IN) => "IN",
+                        _ => "", // unexpected
```

Comment:
Can use `unreachable!()` to panic if it's an unexpected condition.

Reply:
Fixed